### DISCUSSION TITLE
Add the ability to directly plot functions without manual discretization

### DISCUSSION THREAD

**Discussion by musm** [JuliaPlots/Makie.jl#4168] [Ideas]
Add the ability to directly plot functions without manual discretization

**Comment by SimonDanisch** [comment]
isn't that supported? What do you mean exactly?

**Comment by musm** [comment]
`plot(sin)` works using Plots.jl it automatically discetizes the x axis.

**Comment by musm** [comment]
ah I see the correct syntax is 
`lines(0..pi, sin)`

**Comment by asinghvi17** [comment]
Michael linked me the Plots code, I should try it

**Comment by mkborregaard** [comment]
It's just that Plots has a philosophy of always allowing maximum laziness in the user. "Limits, no that's too hard to specify". Surprisingly effective philosophy.

**Comment by ffreyer** [comment]
Are we planning to allow this or can this be closed? Not specifying the domain seems like something that would not produce useful results most of time...
